packages/docs/docs/integration/index.md

```diff
@@ -10,20 +10,21 @@ Medplum's integrations the most commonly used features of the platform, and the
-| [Candid Health](https://www.joincandidhealth.com/)             | Billing         | Revenue cycle and insurance eligibility check | [Candid bot](https://github.com/medplum/medplum/tree/main/examples/medplum-demo-bots/src/candid-health)     |
-| [Particle](https://www.particlehealth.com/)                    | HIE             | Read/Write from Exchanges                     | Open source coming soon                                                                                     |
-| [Epic Systems](https://www.epic.com/)                          | EHR             | Read/Write via FHIR API                       | [Epic JWT authentication](https://github.com/medplum/medplum/tree/main/examples/medplum-demo-bots/src/epic) |
-| [Recaptcha](https://www.google.com/recaptcha/about/)           | Security        | Enable recaptcha on patient registration      | [Setup recaptcha](/docs/auth/custom-emails#setup-recaptcha)                                                 |
-| [Datadog](https://www.datadoghq.com/)                          | Observability   | Application monitoring                        | [Datadog sidecar setup](/docs/self-hosting/datadog)                                                         |
-| [Snowflake](https://www.datadoghq.com/)                        | Data warehouse  | Synchonize data to datawarehouse              | Documentation coming soon                                                                                   |
+| Name                                                                   | Type            | Description                                   | Documentation                                                                                               |
+| ---------------------------------------------------------------------- | --------------- | --------------------------------------------- | ----------------------------------------------------------------------------------------------------------- |
+| [Okta](https://www.okta.com/)                                          | Authentication  | Enable Okta SSO for Providers                 | [Okta setup](/docs/auth/methods/domain-level-identity-providers#okta-setup)                                 |
+| [Auth0](https://auth0.com/)                                            | Authentication  | Enable Auth0 SSO for Providers and Patients   | [Auth0 account Setup](/docs/auth/methods/external-ids#invite-user)                                          |
+| [Google Authentication](https://safety.google/authentication/)         | Authentication  | Enable Google SSO for Providers and Patients  | [Google Auth Setup](/docs/auth/methods/google-auth)                                                         |
+| [Azure SSO](https://learn.microsoft.com/en-us/entra/identity-platform) | Authentication  | Enable Azure SSO for Providers and Patients   | [Azure Auth Setup](https://www.medplum.com/docs/auth/methods/external-identity-providers)                   |
```

Comment:
Fine for now, but I kindly request making these changes in a separate PR in the future.